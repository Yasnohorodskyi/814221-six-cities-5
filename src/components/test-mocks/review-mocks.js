export const review = {
  comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
  date: `2019-05-08T14:13:56.569Z`,
  id: 1,
  rating: 4,
  user: {
    photo: `/img/avatar-angelina.jpg`,
    id: 4,
    isPro: false,
    name: `Lena`,
  },
};

export const reviews = [
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-05-08T14:13:53.569Z`,
    id: 1,
    rating: 4,
    user: {
      photo: `/img/avatar-angelina.jpg`,
      id: 4,
      isPro: false,
      name: `Lena`,
    },
  },

  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-05-08T14:13:56.569Z`,
    id: 2,
    rating: 5,
    user: {
      photo: `/img/avatar-angelina.jpg`,
      id: 34,
      isPro: true,
      name: `Masha`,
    },
  },
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-05-08T14:13:58.569Z`,
    id: 3,
    rating: 3,
    user: {
      photo: `/img/avatar-angelina.jpg`,
      id: 39,
      isPro: false,
      name: `Masha`,
    },
  },
];

export const reviewsBeforeAdapt = [
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-05-08T14:13:56.569Z`,
    id: 2,
    rating: 5,
    user: {
      avatar_url: `/img/avatar-angelina.jpg`,
      id: 34,
      is_pro: true,
      name: `Masha`,
    },
  },
  {
    comment: `A quiet cozy and picturesque that hides behind a a river by the unique lightness of Amsterdam.`,
    date: `2019-05-08T14:13:58.569Z`,
    id: 3,
    rating: 3,
    user: {
      avatar_url: `/img/avatar-angelina.jpg`,
      id: 39,
      is_pro: false,
      name: `Masha`,
    },
  },
];
