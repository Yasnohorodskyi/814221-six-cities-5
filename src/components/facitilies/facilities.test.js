import React from "react";
import renderer from "react-test-renderer";
import {offers} from "../test-mocks/offers-mocks";
import Facilities from "../facitilies/facilities";

describe(`<Facilities /> render`, () => {
  it(`renders correctly with full information`, () => {
    const facilitiesComponent = renderer
      .create(<Facilities facilities={offers[0].facilities} />)
      .toJSON();

    expect(facilitiesComponent).toMatchSnapshot();
  });
});
