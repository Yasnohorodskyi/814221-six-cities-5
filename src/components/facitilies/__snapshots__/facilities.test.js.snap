// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Facilities /> render renders correctly with full information 1`] = `
<ul
  className="property__inside-list"
>
  <li
    className="property__inside-item"
  >
    Laptop friendly workspace
  </li>
  <li
    className="property__inside-item"
  >
    Washer
  </li>
  <li
    className="property__inside-item"
  >
    Towels
  </li>
  <li
    className="property__inside-item"
  >
    Fridge
  </li>
  <li
    className="property__inside-item"
  >
    Air conditioning
  </li>
  <li
    className="property__inside-item"
  >
    Baby seat
  </li>
  <li
    className="property__inside-item"
  >
    Breakfast
  </li>
</ul>
`;
